SS_write_ssnew.tpl

```diff
@@ -1487,15 +1487,15 @@ FUNCTION void write_nucontrol()
   NuStart << F_reporting << " # F_std_reporting_units: 0=skip; 1=exploitation(Bio); 2=exploitation(Num); 3=sum(Apical_F's); 4=mean F for range of ages (numbers weighted); 5=unweighted mean F for range of ages" << endl;
   if (F_reporting == 4 || F_reporting == 5)
   {
-    NuStart << F_reporting_ages << " #_min and max age over which mean F will be calculated, with F=Z-M" << endl;
+    NuStart << F_reporting_ages << " # min and max age over which mean F will be calculated, with F=Z-M" << endl;
   }
   else
   {
     NuStart << "#COND 10 15 #_min and max age over which mean F will be calculated with F_reporting=4 or 5" << endl;
   }
   NuStart << F_std_basis_rd << " # F_std_scaling: 0=no scaling; 1=F/Fspr; 2=F/Fmsy; 3=F/Fbtgt; where F means annual F_std, Fmsy means F_std@msy; values >=11 invoke N multiyr (up to 9!) using 10's digit; >100 invokes log(ratio)" << endl;
   NuStart << double(mcmc_output_detail) + MCMC_bump << " # MCMC output detail: integer part (0=default; 1=adds obj func components; 2= +write_report_for_each_mceval); and decimal part (added to SR_LN(R0) on first call to mcmc)" << endl;
-  NuStart << ALK_tolerance << " # ALK tolerance ***disabled in code (example 0.0001)" << endl;
+  NuStart << ALK_tolerance << " # ALK tolerance ***disabled in code" << endl;
```

Comment:
@Rick-Methot-NOAA so ALK tolerance is a disabled feature? If so I just need to update the documentation [here](https://github.com/nmfs-stock-synthesis/doc/blob/d02f2d4b50023e87c53baae729d115fd62ffdc86/6starter.tex#L242).

Reply:
Yes, it is disabled.  Please update the documentation.